src/Net/TcpSettings.cs

```diff
@@ -31,6 +32,17 @@ public class TcpSettings
         int? sendBufferSize;
         int? sendTimeout;
 
+        /// <summary>
+        /// Specifies the KeepAlive option of a TCP socket.
+        /// Tuple.Item1: KeepAliveTime.
+        /// Tuple.Item2: KeepAliveInterval.
+        /// </summary>
+        public Tuple<ulong, ulong> KeepAlive
```

Comment:
Config names should be explicit. It is not easy to figure out what item1 and item2 mean in a tuple. We can define a TcpKeepAlive class similar to the one you have in comments in the method implementation, or maybe just two properties for the time and interval on TcpSettings.